easybuild/tools/modules.py

```diff
@@ -63,6 +63,8 @@
 # see e.g., https://bugzilla.redhat.com/show_bug.cgi?id=719785
 LD_LIBRARY_PATH = os.getenv('LD_LIBRARY_PATH', '')
 
+LMOD_USER_CACHE_DIR = os.path.join(os.path.expanduser('~'), '.lmod.d', '.cache')
```

Comment:
the cache location should be a class constant of each module tool (class constants should be neutral).